### PR TITLE
Adding trusted-ca-file option to Securing guide

### LINKED ISSUE #1097: Include agent `trusted-ca-file` attribute in securing Sensu guide
Page: https://docs.sensu.io/sensu-go/5.1/guides/securing-sensu/

To do: Add an example for using the `trusted-ca-file` attribute in the agent configuration

Reference: https://docs.sensu.io/sensu-go/5.1/reference/agent/#security-configuration-flags

Note: This applies to Sensu Go 5.1 only.

### PR DESCRIPTION
**Pull request by asachs01**
## Description
Closes #1097 by adding the following copy:
```
It is also possible to provide a trusted CA as part of the agent configuration by passing `--trusted-ca-file` if starting the agent via `sensu-agent start`.

You you may include it as part of the agent configuration in `/etc/sensu/agent.yml` as: 

{{< highlight yaml>}}
trusted-ca-file: "/path/to/trusted-certificate-authorities.pem"
{{< /highlight >}}
```

### CODE REVIEW

File: content/sensu-go/5.0/guides/securing-sensu.md [hunk 1]
```diff
@@ -95,8 +95,15 @@ backend-url:
 
 The agent will then connect Sensu servers over wss. Do note that by changing the configuration to wss, plaintext communication will not be possible.
 
-_NOTE: If creating a Sensu cluster, every cluster member needs to be present in the configuration. See the [Sensu Go clustering guide][2] for more information on how to configure agents for a clustered configuration._
+It is also possible to provide a trusted CA as part of the agent configuration by passing `--trusted-ca-file` if starting the agent via `sensu-agent start`.
+
+You you may include it as part of the agent configuration in `/etc/sensu/agent.yml` as:
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/securing-sensu.md]:
```suggestion
You may include it as part of the agent configuration in `/etc/sensu/agent.yml` as: 
```

**Comment by asachs01** [comment]
@apaskulin just merged those into the PR. Can I get a :+1: ?

File: content/sensu-go/5.1/guides/securing-sensu.md [hunk 2]
```diff
@@ -96,6 +96,14 @@ backend-url:
 
 The agent will then connect Sensu servers over wss. Do note that by changing the configuration to wss, plaintext communication will not be possible.
 
+It is also possible to provide a trusted CA as part of the agent configuration by passing `--trusted-ca-file` if starting the agent via `sensu-agent start`.
+
+You you may include it as part of the agent configuration in `/etc/sensu/agent.yml` as:
```
Comment (apaskulin) [inline content/sensu-go/5.1/guides/securing-sensu.md:101]:
```suggestion
You may include it as part of the agent configuration in `/etc/sensu/agent.yml` as: 
```

**Comment by asachs01** [comment]
rebase made this wonky. going to close it and reopen.

**Pull request closed without merging**